subsys/input/Kconfig

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig INPUT
+	bool "Input"
+	help
+	  Include input subsystem and drivers in the system config
+
+if INPUT
+
+module = INPUT
+module-str = input
+source "subsys/logging/Kconfig.template.log_config"
+
+config INPUT_INIT_PRIORITY
+	int "Input subsystem and drivers init priority"
+	default 90
+	help
+	  Input subsystem and drivers initialization priority.
+
+config INPUT_THREAD
+	bool "Use a dedicated thread to process input events"
+	default y
+
+if INPUT_THREAD
+
+config INPUT_QUEUE_SIZE
+	int "Input queue size"
+	default 16
+
+config INPUT_THREAD_STACK_SIZE
+	int "Input thread stack size"
+	default 512
```

Comment:
missing `help` here (maybe with some details on how to select appropriate values)

Reply:
Good point, added few more options as well, also renamed `INPUT_QUEUE_SIZE` to `INPUT_QUEUE_MAX_MSGS` as that's not in bytes.